### ISSUE TITLE
Add timestamp formatters

### ISSUE THREAD

**Issue by 4Kaylum** [Voxel-Fox-Ltd/VoxelBotUtils#108]
Probably as a function rather than a class, may need to rename TimeValue to something like TimedeltaValue but I really don't want to do that; we'll see how this goes.

Taking suggestions on reasonable implementations for how to make this work. Definite inputs would be a datetime instance and a format, output would be a string. I can think of either helper function with an enum for type, a range of helper functions, or a wrapper class with properties for each format. 

![Screenshot_20210706-030204_Firefox.jpg](https://user-images.githubusercontent.com/5342536/124531795-231a5e00-de07-11eb-8d97-706158eac2c9.jpg)

**Comment by 4Kaylum** [comment]
Here's a basic example I put in the coding channel - this likely won't be something I add until iOS updates to display them properly.

```python
class TimeFormatter(object):
    def __init__(self, time):
        self.time = time
    def __str__(self):
        return self.short_datetime
    @property
    def _format_string(self):
        return f"<t:{int(self.time.timestamp())}:{{}}>"
    @property
    def short_datetime(self):
        return self._format_string.format("f")
    @property
    def long_datetime(self):
        return self._format_string.format("F")


from datetime import datetime as dt, timedelta

t = TimeFormatter(dt.utcnow() - timedelta(days=4))
await channel.send(f"{t}\n{t.short_datetime}\n{t.long_datetime}")
```

**Comment by 4Kaylum** [comment]
Closed with 416b724719090499d797f7eac34d8c050f8ae9d3.
